### ISSUE TITLE
Upload to Opensea not displaying NFT name.

### ISSUE THREAD

**Issue by GregAdkins** [HashLips/hashlips_nft_contract#47]
Upload to Opensea displays token_Id in place of the metadata 'name' attribute. I need the metadata name to display, not token_id.

**Comment by ArabianAce** [comment]
same issue with mine
Did you find out the solution?

**Comment by GregAdkins** [comment]
no, did you have a fix?

**Comment by ArabianAce** [comment]
Yes, I did a mistake in hidden uri metadata file. Had a comma at the end of Image URI mentioned in the meta data file. Removed the comma, uploaded the metadata file again on pinata, and written the hiddenmetadata uri inside ether scan. Waited 2 to 3 minutes and all set.

**Comment by GregAdkins** [comment]
Thanks my upload is OK. I'm having a problem with gas limits on polygon.
thanks!
